src/main/java/lotto/LottoWinner.java

```diff
@@ -0,0 +1,36 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class LottoWinner {
+    private final Lotto winnerLotto;
+    private final LottoNumber bonusLottoNumber;
+
+    private LottoWinner(LottoNumber[] winnerLotto, LottoNumber bonusLottoNumber) {
+        this.winnerLotto = Lotto.of(winnerLotto);
+        this.bonusLottoNumber = bonusLottoNumber;
+    }
+
+    public static LottoWinner of(LottoNumber[] winnerLotto, LottoNumber bonusLottoNumber){
+        return new LottoWinner(winnerLotto, bonusLottoNumber);
+    }
+
+
+
+    public List<Rank> matchingWinnerNumber(LottoTicket lottoTicket){
+        List<Rank> lottoRankList = new ArrayList<>();
+        for(Lotto lotto : lottoTicket.getLottoList()){
```

Comment:
```suggestion
        for (Lotto lotto : lottoTicket.getLottoList()) {
```
코드컨벤션 공백부분 지켜주시면 좋을 것 같아요 🙂